brain/compartmentReportView.h

```diff
@@ -78,6 +78,22 @@ class CompartmentReportView
      */
     BRAIN_API std::future<brion::Frames> load(double start, double end);
 
+    /** Load frames between start and end time stamps.
+     *
+     * @param start the start time stamp
+     * @param end the end time stamp
+     * @param stride the time stride
```

Comment:
In my case I converted time_start, time_end and time_step to integers prior to passing it to C++ reader, so that one was working with integers only.